File: src/com/leetcode/Easy/CrawlerLogFolder.java
package com.leetcode.Easy;

import java.util.Stack;

// 1598. Crawler Log Folder
public class CrawlerLogFolder {
    public int minOperations(String[] logs) {
        Stack<String> stack = new Stack<>();
        for (String log : logs) {
            if(log.equals("../")) {
                if(!stack.isEmpty())
                    stack.pop();
            } else if(log.equals("./")) {
            } else {
                stack.push(log);
            }
        }
        return stack.size();
    }
}

File: JavaScript/Easy/CrawlerLogFolder.js
/**
 * 1598. Crawler Log Folder
 * @param {string[]} logs
 * @return {number}
 */
const minOperations = function (logs) {
  let stack = new Array();
  logs.forEach((log) => {
    if (log === "../") {
      if (stack.length > 0)
        stack.pop();
    } else if (log === "./") {

    } else {
      stack.push(log);
    }
  });

  return stack.length;
};